### ISSUE TITLE
plugin not loading devices on network

### ISSUE THREAD

**Issue by allenlbrown** [joeyhage/homebridge-alexa-smarthome#119]
**Describe Your Problem:**

Plugin stopped working. The Amazon token expired and was able to update that. I didn't include any devices so should load all (there are only two switches). Getting an error message now:

After initialization - InvalidResponse(Invalid list of Alexa devices found for the current Alexa account: {
  "success": true
})

**Logs:**

```
After initialization - InvalidResponse(Invalid list of Alexa devices found for the current Alexa account: {
  "success": true
})```

**Plugin Config:**

```json
{
    "devices": [],
    "excludeDevices": [],
    "auth": {
        "refreshInterval": 4,
        "proxy": {
            "clientHost": "http://192.168.0.208",
            "port": 7000
        }
    },
    "amazonDomain": "amazon.com",
    "language": "en-US",
    "performance": {
        "cacheTTL": 60,
        "backgroundRefresh": true
    },
    "debug": false,
    "platform": "HomebridgeAlexaSmartHome"
}```

**Screenshots:**

<!-- If applicable, add screenshots to help explain your problem. -->

**Environment:**

- **Plugin Version**:
- **Homebridge Version**: <!-- homebridge -V -->
- **Node.js Version**: <!-- node -v -->
- **NPM Version**: <!-- npm -v -->
- **Operating System**: <!-- Raspbian / Ubuntu / Debian / Windows / macOS / Docker / hb-service -->

<!-- Click the "Preview" tab before you submit to ensure the formatting is correct. -->

**Comment by hkosbab** [comment]
have the same issue with my echo dot and fire tv. Although I've setup everything according to description.

**Comment by joeyhage** [comment]
This should be fixed as of v2.2.0
